docs/user-guide/pods/index.md

```diff
@@ -131,42 +115,25 @@ That approach would provide co-location, but would not provide most of the
 benefits of pods, such as resource sharing, IPC, guaranteed fate sharing, and
 simplified management.
 
-## Durability of pods (or lack thereof)
```

Comment:
Actually, the first 2 paragraphs are probably important to keep here, since they address how pods are intended to be used.